### PR TITLE
chat inbox styling

### PR DESCRIPTION
**Pull request by katiyelae**
This push includes, 
- styling for the inbox
- onClick for when clicking on chat, will take to ('/chat')
- styling for the /chat page 

Needs finished, 
- styling the header. The user image placeholder doesn't seem to be moving like the other items in the flex box cause the alignment to be off. 
- scroll feature should be added 

Screenshot 2024-02-20 at 11 00 26 AM
Screenshot 2024-02-20 at 11 00 51 AM

### CODE REVIEW

File: src/components/ChatFeature/Chats/Chats.jsx
Comment (Ant-Shell) [inline src/components/ChatFeature/Chats/Chats.jsx:1]: Looks good - don't forget the Link to the `/chat` route
```
import { Link } from "react-router-dom";

const Chats = () => {
    return(
        <Link to={'/chat'}>
            <div className="chats">
                <div className="userChat">
                        <img src={User} alt="user" className="userImg" />
                        <div className="userChatInfo">
                            <span>Jane</span>
                            <p>Look forward to meeting you!</p>
                        </div>
                        <div className="dateTime">
                        <span>12/2/24</span>
                        </div>
                </div>
            </div>
        </Link>
    )
}
```

Comment (katiyelae) [reply src/components/ChatFeature/Chats/Chats.jsx:1]: Just added and pushed it. Going to do that for the back button as well

**Review by Ant-Shell — APPROVED** [review]
Looks good, overall. Approving.

**Comment by Ant-Shell** [comment]
Just a heads-up; what's causing that extra space at the top of the page is because of Line 8 in `ChatInbox.jsx`

```
const ChatInbox = () => {
    return(
        <div className="home">  <--- Line 8
            <div className="container">
                <Sidebar />
                <Navbar />
            </div>
        </div>
    )
}
```

Renaming it to something like `<div className="chat-inbox">` should fix it.

The className `.home` is also used in `HomePage.scss`, in various places, such as lines 26 - 41, which adds some unwanted styling in this component (I assume).

**Comment by katiyelae** [comment]
> Just a heads-up; what's causing that extra space at the top of the page is because of Line 8 in `ChatInbox.jsx`
> 
> ```
> const ChatInbox = () => {
>     return(
>         <div className="home">  <--- Line 8
>             <div className="container">
>                 <Sidebar />
>                 <Navbar />
>             </div>
>         </div>
>     )
> }
> ```
> 
> Renaming it to something like `<div className="chat-inbox">` should fix it.
> 
> The className `.home` is also used in `HomePage.scss`, in various places, such as lines 26 - 41, which adds some unwanted styling in this component (I assume).


Thanks for helping figure that out for me! I went ahead and fixed it and will push it with any other updates made today

**Pull request closed without merging**